com.reprezen.swagedit.openapi3/src/com/reprezen/swagedit/openapi3/hyperlinks/LinkOperationHyperlinkDetector.java

```diff
@@ -0,0 +1,52 @@
+package com.reprezen.swagedit.openapi3.hyperlinks;
+
+import java.util.Iterator;
+import java.util.List;
+import java.util.Objects;
+
+import org.eclipse.jface.text.IRegion;
+import org.eclipse.jface.text.ITextViewer;
+import org.eclipse.jface.text.hyperlink.IHyperlink;
+
+import com.fasterxml.jackson.core.JsonPointer;
+import com.reprezen.swagedit.core.editor.JsonDocument;
+import com.reprezen.swagedit.core.hyperlinks.AbstractJsonHyperlinkDetector;
+import com.reprezen.swagedit.core.hyperlinks.SwaggerHyperlink;
+import com.reprezen.swagedit.core.model.AbstractNode;
+import com.reprezen.swagedit.core.model.Model;
+
+public class LinkOperationHyperlinkDetector extends AbstractJsonHyperlinkDetector {
+
+    @Override
+    protected boolean canDetect(JsonPointer pointer) {
+        return pointer != null && pointer.toString().matches("/components/links/(\\w+)/operationId");
+    }
```

Comment:
This regex only works when a link is defined in `components`, but it can also be defined directly in a response:
```yaml
  /2.0/repositories/{username}/{slug}: 
    get: 
      operationId: getRepository
      responses: 
        '200':
          description: The repository
          links:
            repositoryPullRequests:
              operationId: getRepository
```
In that case, navigation does not work.